ocw/lib/EC2.py

```diff
@@ -89,45 +88,33 @@ def all_clusters(self):
         return clusters
 
     @staticmethod
-    def needs_to_delete_snapshot(snapshot, cleanup_ec2_max_snapshot_age_days) -> bool:
-        delete_older_than = date.today() - timedelta(days=cleanup_ec2_max_snapshot_age_days)
-        if datetime.date(snapshot['StartTime']) < delete_older_than:
-            regexes = [
-                re.compile(r'''^OpenQA upload image$'''),
-                re.compile(r'''^Created by CreateImage\([\w-]+\) for ami-\w+ from vol-\w+$''')
-            ]
-            for regex in regexes:
-                m = re.match(regex, snapshot['Description'].strip())
-                if m:
-                    return True
-        return False
+    def needs_to_delete_by_age(creation_time, cleanup_ec2_max_age_days) -> bool:
```

Comment:
```suggestion
    def is_outdated(creation_time, cleanup_ec2_max_age_days) -> bool:
```

Perhaps make it clearer, that this is a comparison method

Reply:
done